esp-hal/README.md

```diff
@@ -46,6 +46,12 @@ For help getting started with this HAL, please refer to [The Rust on ESP Book] a
 [s2-trm]: https://www.espressif.com/sites/default/files/documentation/esp32-s2_technical_reference_manual_en.pdf
 [s3-trm]: https://www.espressif.com/sites/default/files/documentation/esp32-s3_technical_reference_manual_en.pdf
 
+## `unstable` feature
+
+We aim to avoid enabling experimental features automatically. Allowing automatic access could result in users unintentionally utilizing APIs that are still under development.
```

Comment:
this is very brief and fuzzy at the same time - we should mention that the stable feature set is ... stable - and other parts guarded by the unstable feature will likely see breaking changes .... something like that

I don't think we need to explain here why we don't enable unstable by default 🤔